index.js

```diff
@@ -273,29 +360,70 @@ class GoogleAuth extends DbBase {
     return data && this.checkAdmAccessLevel(data.username, level)
   }
 
-  _getOAuth2Client (redirectUriKey = undefined) {
-    const { clientId, clientSecret, redirectUris } = this.conf.google
+  /**
+   * Create OAuth2 client for Google authentication
+   * Note: redirectUris are optional for mobile clients (they use ID tokens, not OAuth code flow)
+   * @param {Object} opts
+   * @param {string|undefined} opts.clientKey - Client key hint
+   * @param {string|undefined} opts.redirectUriKey - Redirect URI key (e.g., 'ssoAuth') - only needed for web OAuth flows
+   * @param {string|undefined} opts.tokenAud - Token audience for validation
+   * @returns {google.auth.OAuth2}
+   */
+  _getOAuth2Client ({ clientKey, redirectUriKey, tokenAud } = {}) {
+    const { clientId, clientSecret, redirectUris } = this._resolveGoogleClient({
```

Comment:
since for mobile we won't have clientSecret and redirectUris will it still work?